admin_guide/out_of_resource_handling.adoc

```diff
@@ -387,14 +587,14 @@ To reach that capacity, either some pod is using more than its request, or the
 system is using more than `1Gi`.
 
 If a node has 10 Gi of capacity, and you want to reserve 10% of that capacity for
-the system daemons, do the following:
+the system daemons (`system-reserved`), perform the following calculation:
 
 ----
 capacity = 10 Gi
 system-reserved = 10 Gi * .01 = 1 Gi
```

Comment:
@mburke5678 It's my first time to see this format. A new thing for me, thanks. Since it's a universal writing, I can accept it.